discovery/syncer.go

```diff
@@ -426,19 +442,32 @@ func (g *gossipSyncer) channelGraphSyncer() {
 			// an ending reply, or just another query from the
 			// remote peer.
 			select {
-			case msg := <-g.gossipMsgs:
+			case msg := <-g.gossipMsgs.ChanOut():
+				// Ensure that this is at least an
+				// lnwire.Message, otherwise, we can't properly
+				// manipulate it.
+				netMsg, ok := msg.(lnwire.Message)
```

Comment:
I think we should really type assert without checking `ok` here, as anything not `lnwire.Message` shouldn't be on this channel.